compiler/test/dotty/tools/dotc/CompilationTests.scala

```diff
@@ -42,6 +42,7 @@ class CompilationTests extends ParallelTesting {
     compileList("compileStdLib", TestSources.stdLibWhitelisted, scala2Mode.and("-migration", "-Yno-inline")) +
     compileFile("tests/pos/nullarify.scala", defaultOptions.and("-Ycheck:nullarify")) +
     compileFile("tests/pos-scala2/rewrites.scala", scala2Mode.and("-rewrite")).copyToTarget() +
+    compileFile("tests/pos-special/i4785.scala", defaultOptions.and("-Yno-imports")) +
```

Comment:
You can achieve the same effect by importing explicitly anything from Predef. Then you could make it a regular test and would not need pos-special.